test/features/legend.test.ts

```diff
@@ -0,0 +1,41 @@
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS, fakeTextShort } from '../factories';
+
+async function expectEditExpect(tester: any, toContain: string) {
+  expect(tester.text()).toContain(toContain);
+  expect(tester.find('input').length > 0).toBe(false);
+
+  tester.find(`button.btn-edit`).simulate('click');
+  await tester.refresh();
+
+  expect(tester.text()).toContain(toContain);
+  expect(tester.find('input').length > 0).toBe(true);
```

Comment:
Similar to above, any reason not to make this `expect(tester.find('input').length).not.toBe(0)` ?

Reply:
Good point, much cleaner!